Dicey Combat.md

```diff
@@ -0,0 +1,112 @@
+  - [Version History](#version-history)
+    - [Background asset](#background-asset)
+    - [Dice asset](#dice-asset)
+  - [Further Plan](#further-plan)
+    
+
+#
+
+## Game Concept
+Dicey Combat is a 2D arcade game for 2 players.  
+
+### Introduction
+
+In the game, players roll two dice to move forward. Players conquer the opponents’ region and eat the token. Each player has four lives for a round. The game will end when one player loses all of the 4 lives. 
+
+### Specification
+* Handle 2 player input in one screen 
+* Developed using C++ on Splashkit 
+* On arcade machine, player controls one button for rolling dice, one button for moving game characters, one button for restart the game 
+
+#
+
+## User Stories
+* As a student I want a machine with the ability to play my game creations so that I can showcase and share them with my peers.
+* As a student I want to be able to have a guide so I can figure out how to use the machine and how I can get my game on the machine
+* As a creator I want to know that I can safely upload my games without them being stolen or re uploaded to other sources so that I can continue to have it as an individual title.
+* As a creator I want a server with the capability to update and deploy iterations of my game so that I can showcase it
+
+#
+
+
+## Procedures
+Read more on :
+[Trello Board](https://trello.com/b/ZqLqB2VB/cool-game-team)
+
+* Writing a step-by-step user guide for git pull request 
+
+* Planning, design and implementation  
+
+* Draft plan of the design and code implementation 
+
+* Researching on image assets 
+
+* Sketching design  
+
+* Low fidelity representation of the game (using simple shapes on powerpoint) 
+
+* Upskill on Procreate (graphic editor application) 
+
+* Creating image assets 
+
+  Background 
+
+  Players 
+
+  Roll and Pass buttons 
+
+  Dice 
+
+  Indicators 
+
+* Research on code implementation 
+
+* Write a Pseudocode for the game play 
+
+#
+
+## Version History
+[Time-Lapse of Image Assets](https://youtu.be/5eA3UTkKqgs) 
+
+[Draft prototype presentation of game design](https://docs.google.com/presentation/d/1NAFD1ZbJFT7JtbG1qoaIBiS6UfSF0FfsAPnxu15A-Gk/edit?usp=sharing)  
+### Background asset
+
+![image](Images\Background1.png)
+Figure 1. Draft prototype presentation of game design
+
+![image](Images\DiceyCombatBackground.png)
+Figure 2. Rought draft of the background on PhotoShop 
+
+![image](Images\Background2.png)
+Figure 3. Semi-finished background on Procreate
+
+![image](Images\Background3.png)
+Figure 4. Final version of the background on Procreate 
+
+### Dice asset
+
+![image](Images\Tokens.png)
+Figure 4. Final version of the indicators, dices, and players on Procreate  
+
+![image](Images\Dice1.png)
+Figure 5. Dice version 1
+
+![image](Images\Dice2.png)
+Figure 6. Dice version 2
+
+#
+
+## Further Plan
+In the next trimester, Dicey Combat will be finished and polished. It will be successfully presented on the arcade machine and be played.
```

Comment:
Reword: 
```suggestion
In the next trimester, Dicey Combat will be refined and completed. It will be successfully presented on the arcade machine to be played.
```